content-src/components/StartupOverlay/_StartupOverlay.scss

```diff
@@ -100,7 +100,7 @@
   .extra-links {
     font-size: 12px;
     max-width: 340px;
-    margin: 14px 50px;
+    margin: 17px 50px;
```

Comment:
erica asked me to sneak this change, and the margin change in `.skip-button` into this pr while i'm here

Reply:
can you add that to the commit message